WordPress/src/main/res/layout/base_site_listitem.xml

```diff
@@ -6,6 +6,7 @@
     android:gravity="center_vertical">
 
     <ImageView
+        style="@style/MySiteBlavatarImageView"
```

Comment:
Apparently applying this style here makes the site list icon have a second background. It becomes more apparent when you look at the corners of the icon:
![s1](https://user-images.githubusercontent.com/830056/92062523-50255c80-ed6f-11ea-8877-8f07b4f893e6.png)

Here's what it should look like:
![s2](https://user-images.githubusercontent.com/830056/92060577-8dd3b680-ed6a-11ea-9ef2-75af6323e11e.png)
Maybe this can be fixed by loading the icon image using `BLAVATAR_ROUNDED_CORNERS` [here](https://github.com/wordpress-mobile/WordPress-Android/blob/eebe812bcaa52fdc45fff257f335c9bc9fdb9f32/WordPress/src/main/java/org/wordpress/android/ui/main/MySiteFragment.kt#L991). That would also account for the difference in the background color of the default icon.